src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -234,6 +235,11 @@ public class Feature
         /// </summary>
         public static Feature GeoNearCommand => __geoNearCommand;
 
+        /// <summary>
+        /// Gets the getIndexesContainsNS feature.
+        /// </summary>
+        public static Feature GetIndexesContainsNS => __getIndexesContainsNS;
+
```

Comment:
I don't think we should add a Feature for this.

This is only affecting the Legacy API tests and that is not enough of a reason to add a new public Feature to this class.

Reply:
Done